osu.Game/Rulesets/Mods/ModMuted.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Track;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Audio;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Configuration;
+using osu.Game.Graphics.Containers;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.UI;
+using osu.Game.Skinning;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public abstract class ModMuted : Mod
+    {
+        public override string Name => "Muted";
+        public override string Acronym => "MU";
+        public override IconUsage? Icon => FontAwesome.Solid.VolumeMute;
+        public override string Description => "Who needs music in a rhythm game?";
```

Comment:
This semi-ironic description-as-rhetorical-question is probably my single biggest problem with this mod, conceptually speaking. Like at this point you might as well use the volume meter to completely silence the music, but what's the point? It's a rhythm game.

Reply:
Might have gone a bit overboard there. How's the new description looking?